CrowsZA/functions/fn_fireSupport.sqf

```diff
@@ -0,0 +1,205 @@
+/*/////////////////////////////////////////////////
+Author: Radkewolf
+			   
+File: fn_fireSupport.sqf
+Parameters: pos
+Return: none
+
+Sets off firesupport in the targeted area
+
+*///////////////////////////////////////////////
+params [["_pos",[0,0,0],[[]],3], ["_logic",objNull,[objNull]]];
+
+// Validate module target
+scopeName "Main";
+private _fnc_errorAndClose = {
+    params ["_msg"];
+    diag_log _msg;
+    breakOut "Main";
+};
+
+// open dialog
+//ZEN
+private _onConfirm =
+{
+	params ["_dialogResult","_in"];
+	_dialogResult params
+	[
+		"_type",
+		"_customType",
+		"_radius",
+		"_seconds",
+		"_delay",
+		"_salvos",
+		"_guns"
+	];
+
+	//Get in params again
+	_in params [["_pos",[0,0,0],[[]],3], ["_logic",objNull,[objNull]]];
+
+	//Check if placed on an object
+	_create = isNull _logic;
+
+	//Check if object is firesupport module
+	if(!_create)
+	then
+	{
+		if(!(_logic isKindOf "Logic"))
+		then
+		{
+			_create = true;
+		}
+		else {
+			_mod = _logic getVariable ["_module", ""];
+			if(_mod != "firesupport")
+			then
+			{
+				_create = true;
+			};
+		}
+	};
+
+	//Create new module if not placed on existing one
+	if(_create)
+	then{
+		_logicCenter = createCenter sideLogic;
+		_logicGroup = createGroup _logicCenter;
+		_logic = _logicGroup createUnit ["Logic", _pos, [], 0, "NONE"];
+		_logic setVariable ["_module", "firesupport"];
+		["zen_common_addObjects", [[_logic], objNull]] call CBA_fnc_serverEvent;
+	};
+
+	//Use specific cfgAmmo if filled
+	if(_customType != "")
+	then{
+		_type = _customType;
+	};
+	diag_log "guns:";
+	diag_log str _guns;
+	//Set variables to logic gameobject
+	_logic setVariable ["_type", _type];
+	_logic setVariable ["_radius", _radius];
+	_logic setVariable ["_seconds", _seconds];
+	_logic setVariable ["_salvos", round _salvos];
+	_logic setVariable ["_guns", round _guns];
```

Comment:
should the module be editable by another zeus? In that case I think you might need to add the public flag to the variable so it gets sync'ed across clients. 
So you can just add a ``_logic setVariable ["_type", _type, true];`` to each, then it gets synced globally. 

![image](https://user-images.githubusercontent.com/7889925/123560731-53795100-d7a4-11eb-9779-3717ce923be8.png)

Reply:
Should be, was unaware of that parameter.

Reply:
no worries. locality and multiplayer scripting took me quite some time to get my head around xD